src/cpp/src/image_generation/stable_diffusion_3_pipeline.hpp

```diff
@@ -84,21 +87,21 @@ class StableDiffusion3Pipeline : public DiffusionPipeline {
         if (text_encoder_3 == "T5EncoderModel") {
             m_t5_text_encoder = std::make_shared<T5EncoderModel>(root_dir / "text_encoder_3");
         } else {
-            m_t5_text_encoder = nullptr;
+            OPENVINO_THROW("Unsupported '", text_encoder_3, "' text encoder type");
```

Comment:
It was done in case the model doesn't have text_encoder_3. Not sure if we have such models

https://github.com/huggingface/diffusers/blob/c9e4fab42ca481fe8e0d2456b54ec900fb57730d/src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py#L224

Reply:
handled case when `text_encoder_3` is absent and `null` is used
![{09A0B80D-7591-4342-A9E7-CF2065F52CD2}](https://github.com/user-attachments/assets/a5c127dd-ae95-409f-bdb5-9ccae498f627)